### ISSUE TITLE
local environment variables in netlify.toml, with optional context flag

### ISSUE THREAD

**Issue by sw-yx** [netlify/netlify-dev-plugin#134]
@davidwells already has a prototype of this. 

related: 

- https://github.com/netlify/netlify-dev-plugin/issues/114
- https://github.com/netlify/netlify-dev-plugin/issues/60
- also raised in community

**Comment by iamskok** [comment]
Is there any more progress? I’d love to see this feature implemented 🥳

**Comment by dan-dr** [comment]
yeah hope it doesn't sound too harsh but `netlify dev` is pretty useless in my case if I have to use the admin UI to change environment variables.

Another way to go about it is to check if the environment variables exist already and only use the admin UI ones if they don't.
